docs/docker.rst

```diff
@@ -0,0 +1,92 @@
+=============
+Docker
+=============
+
+.. highlight:: sh
+
+OpenTAXII can also be run using docker. This guide assumes that you have access to a local or remote docker server, and won't go into the setup of docker.
+
+To get a default (development) instance using docker
+
+
+.. code-block:: shell
+
+    docker run -d -p 9000:9000 intelworks/opentaxii
+
+.. note::
+
+    OpenTAXII is now accessible through port 9000, and is not configured.
```

Comment:
In the examples it has that. But here it would make sense to add as well. But then leave out a user?